modelscan/modelscan.py

```diff
@@ -43,7 +44,11 @@ def scan_path(self, path: Path) -> None:
         if path.is_dir():
             self._scan_directory(path)
         elif _is_zipfile(path) or path.suffix in self._supported_zip_extensions():
-            self._scan_zip(path)
+            is_keras_file = True if path.suffix == ".keras" else False
```

Comment:
Probably want to have this as "path.suffix in KerasScan.supported_extensions()" so the extension is only hard-coded in one place

Reply:
Thank you, I have updated it